code/game/gamemodes/cult/items/tome.dm

```diff
@@ -8,6 +8,70 @@
 	w_class = 2.0
 	unique = TRUE
 	slot_flags = SLOT_BELT
+	var/obj/item/inserted_item  //chalk code graciously stolen from the PDA's pen code   -MalMalmulam
```

Comment:
similiarly, all we need to do is this here:
`var/obj/item/pen/drafting/inserted_item = /obj/item/pen/drafting`
then, later:
`inserted_item = new(src)`
and then all we have to check is if it exists, and to make sure that we can ONLY put the chalk in there

Reply:
Done.  List eliminated.  I thought having a list would be weird for this use case too, but I didn't want to mess with the copy-pasted code in case I broke something.